examples/example.bf

```diff
@@ -0,0 +1 @@
+++++++++++[>+++++++>++++++++++>+++>+<<<<-]>++.>+.+++++++..+++.>++.<<+++++++++++++++.>.+++.------.--------.>+.>.
```

Comment:
this file is unused. Why is this needed?

Reply:
in branch "read_from_file_feature" i've added feature that reads input from this file